### PR TITLE
feat: added resolution tolerance + tests (MAPCO-4096)

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -157,5 +157,9 @@
   "forbiddenTypesForParallelIngesion": {
     "__name": "FORBIDDEN_TYPES_FOR_PARALLEL_INGESTION",
     "__format": "json"
+  },
+  "resolutionTolerance": {
```
Comment (asafMasa) [inline config/custom-environment-variables.json]: resolutionTolerance -> resolutionFixedPointTolerance

File: config/default.json [hunk 2]
```diff
@@ -100,5 +100,6 @@
   "forbiddenTypesForParallelIngesion": [
     "Ingestion_New",
     "Ingestion_Update"
-  ]
+  ],
+  "resolutionTolerance": 12
```
Comment (asafMasa) [inline config/default.json]: move to "validationValuesByInfo" section

File: src/utils/pixelSizeValidate.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import config from 'config';
+
+export const pixelSizeValidate = (requestedPixelSize: number, sourcePixelSize: number): boolean => {
+  const resolutionTolerance = config.get<number>('resolutionTolerance');
```
Comment (asafMasa) [inline src/utils/pixelSizeValidate.ts]: move to function attributes

File: src/utils/pixelSizeValidate.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import config from 'config';
+
+export const pixelSizeValidate = (requestedPixelSize: number, sourcePixelSize: number): boolean => {
```
Comment (asafMasa) [inline src/utils/pixelSizeValidate.ts]: pixelSizeValidate -> isPixelSizeValid

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: config/custom-environment-variables.json (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply config/custom-environment-variables.json]: fixed

File: config/default.json (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply config/default.json]: fixed

File: src/utils/pixelSizeValidate.ts (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply src/utils/pixelSizeValidate.ts]: fixed

File: src/utils/pixelSizeValidate.ts (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply src/utils/pixelSizeValidate.ts:4]: its arrow function, we talked about it before with other's PR's  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

**Review by asafMasa — COMMENTED** [review]
only one comment left

File: src/utils/pixelSizeValidate.ts (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply src/utils/pixelSizeValidate.ts]: fixed

**Review by asafMasa — APPROVED** [review]

File: src/utils/pixelSizeValidate.ts (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply src/utils/pixelSizeValidate.ts]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**